From c0730f186dadf5ea52376c96a5216ed894d99d10 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Wed, 23 Jan 2019 16:55:08 +0000
Subject: [PATCH 0866/1000] MANTIS 0053235 New Baggage-Handling status
 BAGGAGE_INTERFERENCE_STAFF codes.idl V1.4.3 - added statuscodes
 BAGGAGE_INTERFERENCE_STAFF

git-svn-id: svn://localhost/SelfServiceCommon/trunk@1428 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Cuss/codes.idl | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/SelfServiceCommon/Massai/idl/Cuss/codes.idl b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
index 028c2146..b4ff3077 100644
--- a/SelfServiceCommon/Massai/idl/Cuss/codes.idl
+++ b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
@@ -6,7 +6,7 @@
 //
 //  Date:      31.03.2015
 //
-//  Version:   1.4.2
+//  Version:   1.4.3
 //
 //  Author:    IATA Passenger Experience Management Group: CUWG CUSS-TSG
 //
@@ -78,6 +78,11 @@ Changed:   Added baggage handling status codes (Nicolas Bruns, 29.06.2004)
 @li Mantis 0045822: MR-Component: VisionBox: PPR, Face; HRS: Iris
 - added extended datastatus DS_TYPES_XB*, DS_TYPES_USER_ID for biometrics
 
+@date 23.01.2019
+@li V1.4.3
+@li Mantis 0053235: New Baggage-Handling status BAGGAGE_INTERFERENCE_STAFF
+- added statuscodes BAGGAGE_INTERFERENCE_STAFF
+
 */
 
 #ifndef CODES_IDL
@@ -356,6 +361,7 @@ module statuscodes
     const long BAGGAGE_INTERFERENCE_USER     = 149; /**< <b>Scope:</b> private + platform  */
     const long BAGGAGE_INTRUSION_SAFETY      = 150; /**< <b>Scope:</b> private + platform  */
     const long BAGGAGE_NOT_CONVEYABLE        = 151; /**< <b>Scope:</b> private + platform  @note Specific reason codes are reported to the application within the XML data */
+    const long BAGGAGE_INTERFERENCE_STAFF    = 152; /**< <b>Scope:</b> private + platform; materna den extension */
     const long BAGGAGE_INCORRECTLY_PLACED    = 170; /**< <b>Scope:</b> private + platform; materna ccm extension */
     const long BAGGAGE_TUB_REQUIRED          = 171; /**< <b>Scope:</b> private + platform; materna ccm extension */
     const long BAGGAGE_HANDLE_EXTENDED       = 172; /**< <b>Scope:</b> private + platform; materna ccm extension */
-- 
2.41.0.windows.1

